helpers/middleman/blog.rb

```diff
@@ -3,21 +3,21 @@
 
 module MiddlemanBlogHelpers
 
-  def blog_posts(order: nil)
+  def blog_posts(order: :latest_first)
     @blog_posts ||= {}
     return @blog_posts[order] if @blog_posts[order]
 
     @blog_posts[order] ||= case order
-      when :youngest_first
+      when :oldest_first
         data.blog.posts.values.sort do |a,b|
-          a['created_at'] <=> b['created_at']
+          a['created_at'].to_i <=> b['created_at'].to_i
```

Comment:
I'm surprised. I assume `a['created_at']` is a timestamp, so it should work with `a['created_at'] <=> b['created_at']` already and the `to_i` is not necessary. Am I wrong?

Reply:
You're right.  I tried locally and reverted as it passed without.  In 696dc0b3.

Struggled here because the naming confused me as to why stuff wasn't working, so I think I tried a few things and this got committed…